src/browser/util/encodeArgs.js

```diff
@@ -0,0 +1,30 @@
+import { u, wallet } from '@cityofzion/neon-js';
+import tree from 'switch-tree';
+
+const encodeArgs = (args) => {
+  // assert args is an array
+  if (!Array.isArray(args)) {
+    return args;
+  }
+
+  // For now this takes care of strings and numbers and nested Arrays
+  // Also this shouldn't be used when NEO addresses are passed for contracts using CheckWitness.
+  // Until we determine what a good practice for general encoding addresses is
+  // - using unhexlify + reverseHex + toScriptHash or just reverseHex as is used in this function -
+  // but the former is necessary for contracts using CheckWitness
+  // the latter is used from neon-js guides to checkBalance.
+  // If the former can be used for those as well, this is no problem.
```

Comment:
:neckbeard: Punctuation is a bit inconsistent throughout this comment, it made it a little difficult to scan since it spans so many lines.  I think adding some periods where needed would help. 😄